Proposing new formatting for titles/subtitles

gomezcatalina opened the pull request:
# Proposed Change/Activity

Extracted all titles from the report, and proposing a new structure more standardized. 

# Why is this important

Consistency and organization are key! 

# Additional Context

This update is proposed before @[stfinnis](https://github.com/stfinnis) starts his editorial magic :mage:

************************************************************************************
# Changes proposed below!

# **REPRODUCIBLE REPORTING: ABOUT THIS DATA DISCOVERY TOOL**  

##### *Background*

##### *System*

##### *Species focus*

##### *Disclaimers*

##### *Data Access & Quality Tiers*

*********************************************************************************
# **SEARCH RESULTS: ECOLOGICAL DIMENSION**  

## **INFORMATION FROM THE NATIONAL AQUATIC SPECIES AT RISK GEODATABASE**   

### **DFO Species at Risk distribution (range)**

##### *Area-specific SAR distribution search results*

### **Critical Habitat of Species at Risk** 

##### *Area-specific critical habitat search results*



## **DATASETS INVOLVING MULTIPLE SPECIES**

### **Maritimes Research Vessel (RV) Survey**

##### *Area-specific Maritimes Research Vessel (RV) Survey search results*

### **Passamaquoddy Bay Biodiversity Trawls (PBBT)**

##### *Area-specific search results for the Passamaquoddy Bay Biodiversity Trawls*

### **Inshore Lobster Trawl Survey (ILTS)** 

##### *Area-specific search results for the Lobster Trawl Survey*

### **Industry Survey and Maritimes Fishery Databases**  

#### ***Industry Survey Database (ISDB)***

#### ***The Maritime Fishery Information System (MARFIS)***

##### *Area-specific MARFIS & ISDB search results*

### **Atlantic Sturgeon Foraging Areas in the Upper Bay of Fundy**

##### *Area-specific Atlantic Sturgeon search results*

### **OCEARCH Shark Tracker** 

##### *Area-specific OCEARCH Shark Tracker search results*

### **Ocean Biodiversity Information System (OBIS), the Global Biodiversity Information Facility (GBIF) and iNaturalist**

- Background descriptions for both OBIS and GBIF -- remove this as subtitles
- iNaturalist data -- remove this as subtitles
- Data searches - remove this as subtitles

##### *Area-specific OBIS search results*

##### *Area-specific GBIF search results*

##### *Area-specific iNaturalist search results*

## **DATASETS INVOLVING CETACEAN SPECIES**

### **Whale Sightings Database (WSDB)**

##### *Area-specific WSDB search results*

### **The Whitehead Lab, Dalhousie University**

##### *Area-specific Whitehead Lab search results*

### **North Atlantic Right Whale Consortium (NARWC) Sightings Database**

##### *Area-specific NARWC search results*

### **Species Distribution Models (SDM): Priority Areas to Enhance Monitoring of Cetaceans**

##### *Area-specific SDM cetacean search results*

### **Blue Whale Important Habitat**

##### *Area-specific Blue Whale Important Habitat search results*

### **Northern Bottlenose Whale important habitat in inter-canyon areas on the eastern Scotian Shelf** 

##### *Area-specific Northern Bottlenose Whale Important Habitat in inter-canyon areas on the eastern Scotian Shelf search results*

*********************************************************************************

# **SEARCH RESULTS: AREAS DESIGNATED FOR SPATIAL PLANNING**

### **Ecologically and Biologically Significant Areas (EBSA)**

##### *Area-specific EBSA search results*

### **Draft Conservation Network Design**

##### *Area-specific Draft Conservation Network Design search results*

*********************************************************************************

# **SEARCH RESULTS: BENTHIC HABITAT & SUBSTRATE**

### **Intertidal Vegetation and Rockweed**

##### *Area-specific Intertidal Vegetation and Rockweed search results*  

*********************************************************************************
# **SEARCH RESULTS: HUMAN DIMENSION**  

## **SHIPPING**

##### *Under-construction*


## **LOADING & DISPOSAL**

### **Estimates of anthropogenic nitrogen loading and eutrophication indicators for the Bay of Fundy and Scotian Shelf**

##### *Area-specific nitrogen load results*

### **Man-made solid classification from the Atlantic Shoreline Classification dataset**

##### *Area-specific man-made solid results*

### **Active and Inactive Disposal at Sea Sites**

##### *Area-specific Active and Inactive Disposal at Sea Sites*

### **Clean power generating stations by type in megawatts (MW)**

##### *Area-specific Clean Power Generating Stations*


## **FISHING**

### ***Fishing effort within Significant Benthic Areas in Canada's Atlantic and Eastern Arctic marine waters***

##### *Area-specific fishing effort search results*

### **Inshore Lobster Landings and Fishing Effort**

##### *Area-specific lobster fishing effort results*


## **AQUATIC INVASIVE SPECIES**

### ***Under-construction***

##### *Area-specific results under-construction*


## **LITERATURE REVIEW: THREATS & MITIGATION FOR SPECIES AT RISK**

#### *SARA Recovery Planning Process documents* 


***************************************************************************
# **SEARCH RESULTS: REGIONAL SYNTHESIS**  

### **Multi-Threat Synthesis**

#### *Threat Classification* 

### **Multi-Species Synthesis**

*********************************************************************************

# **CONTRIBUTORS**

*********************************************************************************

# **REFERENCES**    

****************************************************************************

NEW COMPLIMENTARY RMARKDOWNS

****************************************************************************
# **SUPPLEMENTARY MATERIAL 1: Context about the Ocean Biodiversity Information System (OBIS), the Global Biodiversity Information Facility (GBIF), and iNaturalist**

### **Data structure**

### **Disclaimers**

### **Differing research objectives**

### **Observations over land**

### **Duplicate datasets**

### **Additional data**

### **Basis of Record**


****************************************************************************
# **SUPPLEMENTARY MATERIAL 2: LITERATURE REVIEW ON THREATS & MITIGATION FOR SPECIES AT RISK**

### **Key terms and descriptions**

### **Overview of the SARA recovery planning process**

#### *Committee on the Status of Endangered Wildlife in Canada (COSEWIC) Status Reports*

#### *Recovery Potential Assessments (RPA)*

###### *RPA timeline*

###### *Change in RPA procedure over time*

###### *Types of RPAs*

#### *Threat Assessment*

#### *Competent Minister*

#### *Governor in Council*

#### *Recovery Strategy*

#### *Action Plan*

#### *Management Plan*

### **Disclaimers**


****************************************************************************

# TITLES TO DELETE

##### *About this Report*

### **Species at Risk distribution and Critical Habitat data** -- delete?!   

## **THREATS** -- delete?!

## **Cumulative impact mapping**-- delete?!

#### *Interactive Output* --- delete?!

## Miscellaneous -- delete!?


## **Threats Literature Review -- delete!?

*****************************************************************************

# OLD FORMAT


# **REPRODUCIBLE REPORTING**  

#### **About this Report**

##### *Background*

##### *System*

##### *Species focus*

##### *Disclaimers*

##### *Data Access & Quality Tiers*


# **SEARCH RESULTS**  


## **INFORMATION FROM THE NATIONAL AQUATIC SPECIES AT RISK GEODATABASE**   

### **Species at Risk distribution and Critical Habitat data**    

#### ***DFO Species at Risk distribution (range)***

#### ***Area-specific SAR distribution search results***

#### ***Critical Habitat of Species at Risk*** 

#### ***Area-specific critical habitat search results***


## **Species**

### **Maritimes Research Vessel (RV) Survey**

#### ***Area-specific Maritimes Research Vessel (RV) Survey search results***

### **Passamaquoddy Bay Biodiversity Trawls (PBBT)**

#### ***Area-specific search results for the Passamaquoddy Bay Biodiversity Trawls***

### **Inshore Lobster Trawl Survey (ILTS)** 

#### ***Area-specific search results for the Lobster Trawl Survey***

### **Industry Survey and Maritimes Fishery Databases**  

#### ***Industry Survey Database (ISDB)***

#### ***The Maritime Fishery Information System (MARFIS)***

#### ***Area-specific MARFIS & ISDB search results***

### **Atlantic Sturgeon Foraging Areas in the Upper Bay of Fundy **

#### ***Area-specific Atlantic Sturgeon search results:***

### **OCEARCH Shark Tracker** 

#### ***Area-specific OCEARCH Shark Tracker search results***

## **CETACEANS**


### **Whale Sightings Database (WSDB)**

#### ***Area-specific WSDB search results***

### **The Whitehead Lab, Dalhousie University**

#### ***Area-specific Whitehead Lab search results***

### **North Atlantic Right Whale Consortium (NARWC) Sightings Database**

#### ***Area-specific NARWC search results***

### **Species Distribution Models (SDM): Priority Areas to Enhance Monitoring of Cetaceans**

#### ***Area-specific SDM cetacean search results***

### **Blue Whale Important Habitat**

#### ***Area-specific Blue Whale Important Habitat search results***

### **Northern Bottlenose Whale important habitat in inter-canyon areas on the eastern Scotian Shelf** 

#### ***Area-specific Northern Bottlenose Whale Important Habitat in inter-canyon areas on the eastern Scotian Shelf search results***


## **AREAS DESIGNATED FOR SPATIAL PLANNING**

### **Ecologically and Biologically Significant Areas (EBSA)**

#### ***Area-specific EBSA search results***

### **Draft Conservation Network Design**

#### ***Area-specific Draft Conservation Network Design search results***


## **NEW SECTIONS UNDER-DEVELOPMENT OR COMING SOON!**

## Aquatic Invasive Species


## Cumulative impact mapping 

### **Estimates of anthropogenic nitrogen loading and eutrophication indicators for the Bay of Fundy and Scotian Shelf**

#### ***Area-specific nitrogen load results***

### **Man-made solid classification from the Atlantic Shoreline Classification dataset**

#### ***Area-specific man-made solid results***

### **Active and Inactive Disposal at Sea Sites**

#### ***Area-specific  Active and Inactive Disposal at Sea Sites***

### **Clean power generating stations by type in megawatts (MW)**


## Fishing

#### ***Fishing effort within Significant Benthic Areas in Canada's Atlantic and Eastern Arctic marine waters***

#### ***Area-specific fishing effort search results***

### **Inshore Lobster Landings and Fishing Effort**

#### ***Area-specific lobster fishing effort results***


## Threat Classification



## **REGIONAL SYNTHESIS**

### **Multi-Threat Synthesis**

#### *Interactive Output*

### **Multi-Species Synthesis**


# **CONTRIBUTORS**


# **REFERENCES**

stoyelq reviewed (approved):
Looks good to me, I added a commit about where the formatting should be set for each level, feel free to update that if needed.

stfinnis reviewed (commented):
I think some things are still off:

- OBIS/GBIF are removed!! 💔 
- I am still not a fan of the term "Habitat" as the heading for intertidal vegetation/rockweed, since many of the other sections also involve habitat (e.g., Blue Whale Critical Habitat). What are the other datasets this will eventually include? We had it as "Context" before... is there a better title we can use? I'm not giving up on this!! 
- There's a formatting issue with: **Atlantic Sturgeon Foraging Areas in the Upper Bay of Fundy **. The asterisks shouldn't be showing up.
- I don't know if SPECIES INFORMATION FROM MULTIPLE SOURCES makes sense. Should it be "DATASETS INVOLVING MULTIPLE SPECIES" i.e., not just cetaceans? That still might not be the best name.

gomezcatalina commented:
Thank you all for the feedback earlier today. I updated the table of contents and started a new branch to do these updates on Monday. 

- OBIS/GBIF are now included, with the proposed new sibling supplementary material 
- Habitat title updated to be more encompassing
-  I am still not a fan of the term "Habitat" as the heading for intertidal vegetation/rockweed, since many of the other sections 
- Formatting issues with asterisks should be solved 
-  Updated to "DATASETS INVOLVING MULTIPLE SPECIES" 

@stfinnis: please let me know if you would like to see any more updates in the headers before I start the big updated in the new branch. Thank you!

gomezcatalina commented:
Used this formatting to update entire report. Changes to this may trigger another issue (https://github.com/dfo-mar-odis/shinySpatialApp/issues/78).
